examples/react-urql-retry/src/App.js

```diff
@@ -0,0 +1,28 @@
+import { createClient, fetchExchange, Provider } from "urql";
+import { retryExchange } from '@urql/exchange-retry';
+
+import "./App.css";
+import Color from "./pages/Color";
+
+const client = createClient({
+  url: "https://trygql.dev/graphql/intermittent-colors",
+  exchanges: [
+    retryExchange({
+      maxNumberAttempts: 5,
+      retryIf: error => {
+        return Boolean(error && (error.graphQLErrors.length > 0 || error.networkError));
```

Comment:
Just copying this over from your other PR 🙌 

```suggestion
        return error.graphQLErrors.length || error.networkError;
```

which actually can be simplified to `retryIf: () => true` to be fair, but it's still worth having this here for the example. If you wanted to be very precise, you could add:

```suggestion
        return error.graphQLErrors.some(x => x.extensions?.code === 'NO_SOUP') || !!error.networkError;
```